apps/web/src/pages/Communities/ViewCommunityPage/ViewCommunityPage.tsx

```diff
@@ -1,124 +1,248 @@
+    </>
+  );
+};
+
+const ViewCommunity_Query = graphql(/* GraphQL */ `
+  query ViewCommunity($id: UUID!) {
+    community(id: $id) {
+      ...ViewCommunityPage_Community
     }
   }
 `);
 
 const ViewCommunityPage = () => {
   const intl = useIntl();
-  const routes = useRoutes();
 
   const { communityId } = useRequiredParams<RouteParams>("communityId");
   const [{ data, fetching, error }] = useQuery({
     query: ViewCommunity_Query,
     variables: { id: communityId },
   });
 
-  const [{ fetching: isSubmitting }, executeMutation] = useMutation(
-    UpdateCommunity_Mutation,
-  );
-
-  const handleSave = async (input: UpdateCommunityInput) => {
-    return executeMutation({ id: communityId, community: input }).then(
-      (result) => {
-        if (result.data?.updateCommunity) {
-          toast.success(
-            intl.formatMessage({
-              defaultMessage: "Community updated successfully!",
-              id: "8oFk6S",
-              description: "Message displayed after a community is updated",
-            }),
-          );
-          return;
-        }
-        throw new Error("Failed to save community");
-      },
-    );
-  };
-
-  const navigateTo = useReturnPath(routes.communityTable());
-
   const pageTitle = intl.formatMessage({
     defaultMessage: "Community information",
     id: "W0Bh1G",
     description: "Title for community information",
   });
 
+  const { communityName, navigationCrumbs, navTabs } =
+    useOutletContext<ContextType>();
+
   return (
-    <AdminContentWrapper>
+    <>
       <SEO title={pageTitle} />
-      <Pending fetching={fetching} error={error}>
-        {data?.community ? (
-          <CommunitySection
-            initialData={data.community}
-            onUpdate={handleSave}
-            isSubmitting={isSubmitting}
-          />
-        ) : (
-          <NotFound
-            headingMessage={intl.formatMessage(commonMessages.notFound)}
-          >
-            <p>
-              {intl.formatMessage(
-                {
-                  defaultMessage: "Community {communityId} not found.",
-                  id: "TfbBB7",
-                  description: "Message displayed for community not found.",
-                },
-                { communityId },
-              )}
-            </p>
-          </NotFound>
-        )}
-      </Pending>
-      <Separator data-h2-margin="base(x2, 0, 0, 0)" />
-      <p data-h2-margin="base(x2, 0, 0, 0)">
-        <Link mode="solid" color="secondary" href={navigateTo}>
-          {intl.formatMessage({
-            defaultMessage: "Back to communities",
-            id: "3kmwRC",
-            description:
-              "Button text to return to the table of communities page",
-          })}
-        </Link>
-      </p>
-    </AdminContentWrapper>
+      <Hero
+        title={
+          fetching ? intl.formatMessage(commonMessages.loading) : communityName
+        }
+        crumbs={navigationCrumbs}
+        navTabs={navTabs}
+      />
```

Comment:
I fear my intuition was correct. We use the `title` prop here to announce page context changes on navigation. Since this happens immediately on mount, when I navigate to pages with this pattern, all I here is "loading dot dot dot" and nothing after that.

So, it seems to get stuck on loading (even though it is not).

Reply:
Good catch!  I added API wrappers to the two pages that use fetched data for the h1.
[add api wrappers](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12005/commits/31537426b14ec91cdb738f9a54fabba0ccd212be)